src/controllers/revocation.ts

```diff
@@ -255,18 +255,7 @@ export class RevocationController {
 	 *         content:
 	 *           application/json:
 	 *             schema:
-	 *               type: array
-	 *               items:
-	 *                 type: object
-	 *                 properties:
-	 *                   statusListName:
-	 *                     type: string
-	 *                   statusListVersion:
-	 *                     type: string
-	 *                   statusListId:
-	 *                     type: string
-	 *                   statusListNextVersion:
-	 *                     type: string
+	 *               $ref: '#/components/schemas/CredentialStatusSearchResponse'
```

Comment:
I do believe this response is accurate. Could you make the result for a status list search return an array of objects similar to the response that `/create` does? That one is obviously just returning one, this would return the same fields but multiple ones in an array. (Skipping the `created: true` property)

Reply:
Done! ✅